openapi-cli/src/main/java/io/ballerina/openapi/generators/GeneratorConstants.java

```diff
@@ -133,6 +133,14 @@ public String getValue() {
     public static final String RECORD = "record";
     public static final String IDENTIFIER = "identifier";
     public static final String TYPE_NAME = "typeName";
+    public static final String APPLICATION_PREFIX = "application/";
+    public static final String PAYLOAD = "payload";
+    public static final String FORM_URLENCODED = "x-www-form-urlencoded";
+    public static final String OCTET_STREAM = "octet-stream";
+    public static final String TEXT_PREFIX = "text/";
+    public static final String BINARY = "binary";
+    public static final String PDF = "pdf";
```

Comment:
As we have lot of constants defined here, it will be better if we sort them in ascending order

Reply:
As CheckStyle plugin detects required formatting, we could use this as too :slightly_smiling_face: